cpu/cc26x0/include/cpu_conf.h

```diff
@@ -32,10 +32,16 @@
 #include "cc26x0_i2c.h"
 #include "cc26x0_ioc.h"
 #include "cc26x0_prcm.h"
+#include "cc26x0_rfc.h"
+#include "cc26x0_rfc_cmd.h"
+#include "cc26x0_rfc_data.h"
+#include "cc26x0_rfc_ble.h"
+#include "cc26x0_rfc_ieee.h"
```

Comment:
please drop everything apart from cc26x0_rfc.h (have the others included in rfc.h :-)